tower-http/src/metrics/traffic/future.rs

```diff
@@ -0,0 +1,95 @@
+use super::{Callbacks, FailedAt, ResponseBody};
+use crate::classify::{ClassifiedResponse, ClassifyResponse};
+use futures_core::ready;
+use http::Response;
+use http_body::Body;
+use pin_project::pin_project;
+use std::{
+    fmt,
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+    time::Instant,
+};
+
+/// Response future for [`Traffic`].
+///
+/// [`Traffic`]: crate::metrics::Traffic
+#[pin_project]
+pub struct ResponseFuture<F, C, Callbacks, CallbacksData> {
+    #[pin]
+    pub(super) inner: F,
+    pub(super) classifier: Option<C>,
+    pub(super) request_received_at: Instant,
```

Comment:
I'm not seeing a spot where this ever gets used?  Do we need it?  Seems like callers could capture this in their callback data.

Reply:
Good catch! Can indeed be removed.